### PR TITLE
[MM-27231]: e2e for T1880 - deleted token cannot be used to post

### PR DESCRIPTION
**Pull request by ashishbhate**
#### Summary

- Update `apiAccessToken` to return full token response
- Add  `apiRevokeAccessToken` method to revoke token
- cypress test for T1880 - deleted token cannot be used to post

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-27231

### CODE REVIEW

File: e2e/cypress/support/api_commands.js [hunk 1]
```diff
@@ -151,7 +151,27 @@ Cypress.Commands.add('apiAccessToken', (userId, description) => {
...
+    });
+});
+
+/**
+ * Revoke access token
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} tokenId - The token id to revoke
+ * All parameters are required
+ */
+Cypress.Commands.add('apiRevokeAccessToken', (tokenId) => {
```
Comment (josephbaylon) [inline e2e/cypress/support/api_commands.js:164]: Please move `apiRevokeAccessToken` and `apiAccessToken` to `e2e/cypress/support/api/user.js` and also add corresponding entries to `e2e/cypress/support/api/user.d.ts`

File: e2e/cypress/integration/bot_accounts/token_spec.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                cy.get('#sidebarItem_town-square').click({force: true});
+
+                // * Verify bot message
+                cy.get('@botPost').then((postId) => {
+                    cy.get(`#postMessageText_${postId}`).
+                        should('be.visible').
+                        and('have.text', message);
+                });
+
+                // Revoke bot token
```
Comment (josephbaylon) [inline e2e/cypress/integration/bot_accounts/token_spec.js:62]:
```suggestion
                // # Revoke bot token
```

**Review by josephbaylon — CHANGES_REQUESTED** [review]
Hi @ashishbhate Thanks!! Just minor change requests.

**Review by josephbaylon — APPROVED** [review]
LGTM. Awesome work @ashishbhate !! Thanks!

**Review by calebroseland — APPROVED** [review]
LGTM (sorry it took so long, @ashishbhate)

**Comment by josephbaylon** [comment]
@ogi-m I'll go ahead and merge this PR and update the TM4J and JIRA tickets

**Pull request merged**